docs/pages/reference/join-methods.mdx

```diff
@@ -0,0 +1,427 @@
+---
+title: Join Methods and Token Reference
+description: Describes the different ways to configure a node to join the Teleport cluster.
+keywords: [join, joining, token]
+---
+
+<Admonition type="note" title="Prerequisite">
+You must be familiar with the [Teleport Core concepts](../core-concepts.mdx) before reading this page.
+</Admonition>
+
+## Definitions
+
+### Joining
+
+Joining a Teleport cluster is the act of establishing trust between a new node and
+all the existing nodes, already part of the Teleport cluster. This trust is
+established via certificates. Once a node has its certificates, it can interact
+with the other Teleport nodes.
+
+To request its certificates, a node must prove its identity to the Auth service.
+Teleport offers multiple ways for a joining node to prove its authenticity, they
+are called the Join Methods.
+
```

Comment:
I'd add another sentence here before getting any further. Feel free to tweak the wording, but I'm thinking something like this:

> The joining process only happens when a Teleport service starts up
> for the first time. Once the token is exchanged for certificates, those
> certificates are used on all subsequent attempts to connect.